journals/apps/journals/models.py

```diff
@@ -552,6 +554,7 @@ def update_related_objects(self, clear=False):
         self.documents.set(new_docs)  # pylint: disable=no-member
         self.videos.set(new_videos)  # pylint: disable=no-member
         self.images.set(new_images)  # pylint: disable=no-member
+        self.journal_about_page = self.get_journal_about_page()
```

Comment:
If we are going to leave `get_journal_about_page()` untouched then all of the callers who currently call that method should be changed to use JournalPage.journal_about_page instead otherwise there is no benefit

Reply:
might be clearer to have an internal method `_calculate_journal_about_page()` so it's clear it's doing something not just returning `self.journal_about_page`

Reply:
to summarize, proposing the following:
```
self.journal_about_page = self._calculate_journal_about_page()
```
```
def get_journal_about_page() --> def _calculate_journal_about_page()
```
Other references to journal_page.get_journal_about_page() should be replace by journal_page.journal_about_page

Reply:
I like this. Will change.